frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-subscribe.tsx

```diff
@@ -814,6 +881,15 @@ export const OperatorHubSubscribeForm: React.FC<OperatorHubSubscribeFormProps> =
               )}
             </p>
           </Alert>
+        )}{' '}
+        {tokenizedAuth === 'GCP' && showSTSWarn && (
+          <STSWarningAlert
```

Comment:
I know it not new, but lets swap the Alerts above with the new `STSWarningAlert`.